Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,95 @@
+Branch and release maintenance for Kata Containers project
+
+# Overview
+
+As detailed in the release documentation, the Kata Containers project makes use of
+semantic versioning.  A release version is described by MAJOR.MINOR.PATCH.  Early in
+the project, we will introduce many new features which will require a minor version change
+as well as bug fixes.  To facilitate a stable user environment, Kata will begin to provide
+stable branch based releases as well as a master branch release.
+
+## Branch Management
+Kata Containers will maintain two stable release branches in addition to the master branch.
+Once a new MAJOR or MINOR release is created off of master, a new stable branch will be created for
+the prior MAJOR or MINOR release and the older stable branch will no longer be maintained.
+
+A couple of examples are provided to help clarify this process.
+
+### New bug fix introduced
+Consider the following initial state:
+Master 1.3.rc0
+Stable 1.2.0
+Stable 1.1.2
+ 
+A bug fix is submitted against the runtime which does not introduce new inter-component dependencies.
+This fix will be applied to both the master and stable branch, and the new state will be:
+```
+Master: 1.3.rc1 
+Stable: 1.2.1
+Stable: 1.1.3
+```
```

Comment:
I appreciate this isn't as accessible as what you've written but I personally find tables make consuming data like this a lot easier. Something like:

---

Bugfix change:

| Branch | Original version | New version |
|--|--|--|
| `master` | `1.3.rc0` | `1.3.rc1` |
| `stable-1.2` | `1.2.0` | `1.2.1` |
| `stable-1.1` | `1.1.2` | `1.1.3` |